src/librustc_trans/intrinsic.rs

```diff
@@ -85,12 +86,12 @@ fn get_simple_intrinsic(cx: &CodegenCx, name: &str) -> Option<ValueRef> {
 /// Remember to add all intrinsics here, in librustc_typeck/check/mod.rs,
 /// and in libcore/intrinsics.rs; if you need access to any llvm intrinsics,
 /// add them to librustc_trans/trans/context.rs
-pub fn trans_intrinsic_call<'a, 'tcx>(bx: &Builder<'a, 'tcx>,
+pub fn trans_intrinsic_call<'a, 'tcx>(bx: Builder<'a, 'tcx>,
```

Comment:
Why is the signature of this function changed? From the diff it doesn't look like you really need the ownership.

Reply:
Generating a call to the language item will terminate the block being passed in, so using that block further makes no sense. Passing ownership models this pretty well.

Reply:
Is this just a refactoring or necessary for the new `align_offset`? If it's just refactoring I prefer this being separated into its own commit.